meta-genivi-dev/recipes-multimedia/alsa/alsa-state.bbappend

```diff
@@ -1 +1,9 @@
 FILESEXTRAPATHS_append := ":${THISDIR}/${PN}"
+
+SRC_URI_append_m3ulcb = "\
+  file://m3ulcb.state \
+"
+
+do_install_append_m3ulcb() {
+  mv ${D}${localstatedir}/lib/alsa/m3ulcb.state ${D}${localstatedir}/lib/alsa/asound.state
```

Comment:
Please use "install or cp command". if you use "mv" command, it's possible to occur build failure from second build.

ex) if you try to do "install stage" only, the m3ulcb.state is not existed. So failed.